### ISSUE TITLE
Version 0.1.11 update now doesn't recognize `predef` in `.jshintrc`

### ISSUE THREAD

**Issue by CreativLogic** [ember-cli/ember-cli#3135]
After the 0.1.11 update I'm having an issue with the `predef` section of my `.jshintrc` no recognizing my globals.

Normal configuration as follows:   
```json
{
  "predef": [
    "document",
    "window",
    "-Promise",
    "moment",
    "ucwords",
    "jQuery",
    // ...
  ],
  "browser": true,
  // ...
}
```

Getting the typical `... line 167, col 11, 'moment' is not defined.` errors on *build* or *serve* as if they are not in the `predef` section.

**Comment by rlivsey** [comment]
Just ran into this myself after an update, is it not recognising `predef` or is it not loading the `.jshintrc` at all?

**Comment by rlivsey** [comment]
It's an regression in ember-cli-qunit & is fixed in the latest version.

Update package.json to use `ember-cli-qunit: "0.3.1"` and all is well in the world.

**Comment by rlivsey** [comment]
Related issue: https://github.com/ember-cli/ember-cli-qunit/pull/32

**Comment by CreativLogic** [comment]
Confirmed. `ember-cli-qunit: "0.3.1` fixes.

Sorry for the false alarm. Missed that one.

**Comment by rwjblue** [comment]
> Sorry for the false alarm. Missed that one.

No worries!! The version of ember-cli-qunit shipped with 0.1.10 and 0.1.11 (ember-cli-qunit@0.3.0) did have an issue and is updated in 0.3.1.
